modules/FiniteSetsExt.tla

```diff
@@ -54,4 +54,32 @@ kSubset(k, S) ==
    (*************************************************************************)
    { s \in SUBSET S : Cardinality(s) = k }
 
+-----------------------------------------------------------------------------
+
+(***************************************************************************)
+(* We define Max(S) and Min(S) to be the maximum and minimum,              *)
+(* respectively, of a finite, non-empty set S of integers.                 *)
+(***************************************************************************)
+Max(S) == CHOOSE x \in S : \A y \in S : x >= y
+Min(S) == CHOOSE x \in S : \A y \in S : x =< y
+
+-----------------------------------------------------------------------------
+
+(***************************************************************************) 
+(* If Sets is a set of (non-empty) sets then Choice(Sets) is the set of    *)
+(* all choice functions over Sets, that is, functions that associate some  *)
+(* with every set in Sets.                                                 *) 
+(***************************************************************************) 
+Choice(Sets) == { f \in [Sets -> UNION Sets] : \A S \in Sets : f[S] \in S }
+
+-----------------------------------------------------------------------------
+
+Range(f) == {f[x] : x \in DOMAIN f}
```

Comment:
An operator `Range` is defined in the module `Functions` of the TLAPS library: https://github.com/tlaplus/tlapm/blob/8c811c05e3ccaa12a1dbbad85e76b5e16aed3d0c/library/Functions.tla#L20 .

Reply:
Okay, so if I remove `Range` and add `Functions` to `EXTENDS` will this be acceptable solution?

Reply:
A copy of TLAPS's Functions.tla is already  part of CommunityModules:

https://github.com/tlaplus/CommunityModules/blob/2ebcbd910bcf5a2cc7f62cdb3a11c9e2f2d15641/modules/Functions.tla#L15-L20